SU-SWS/decanter
# Template files name change

# READY FOR REVIEW

# Summary
- Removed ".html" from all Twig and JSON file names
- Updated Grunt file to compile files at correct path
- Note on why we're doing this: we're running into file name collisions. Drupal looks for `something.html.twig`, so are changing everything in Decanter to `something.twig` (removing the `.html` part)

# Needed By (Date)
- Indirectly related to work in Stanford Basic Drupal theme, so sooner the better (but no immediate rush)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that all `.html.twig` and `.html.json` files are now `.twig` and `.json` files, respectively

# Affected Projects or Products
- Decanter
- Stanford Basic

# Associated Issues and/or People
- N/A

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
